psi4/src/psi4/libscf_solver/CMakeLists.txt

```diff
@@ -9,10 +9,11 @@ set(sources_list uhf.cc
                  stability.cc
 )
 psi4_add_module(lib scf_solver sources_list mints diis efp_solver fock)
-if(TARGET PCMSolver::PCMSolver)
+if(TARGET PCMSolver::pcm)
```

Comment:
I made a bunch of commits on the web interface and accidentally clogged both Travis and Distelli...